static/css/MarkerCluster.Default.css

```diff
@@ -0,0 +1,60 @@
+.marker-cluster-small {
```

Comment:
I wonder about the file names. Could you use snake-case (all lowercase and underscores) or is this a pre-defined filename format that Leaflet uses?